.babelrc

```diff
@@ -3,7 +3,8 @@
     ["@babel/env", {
       "targets": {
         "browsers": ["last 2 versions"]
-      }
+      },
+      "useBuiltIns": "usage"
```

Comment:
Can you do some investigating around `useBuiltIns: "usage"` and check to see if `@babel/plugin-transform-runtime` is still required? I _believe_ this supersedes it.

Reply:
Yeah looks like it runs fine without `@babel/plugin-transform-runtime`